elftools/dwarf/dwarfinfo.py

```diff
@@ -502,3 +515,17 @@ def replace_value(data, content_type, replacer):
             structs=structs,
             program_start_offset=self.debug_line_sec.stream.tell(),
             program_end_offset=end_offset)
+
+    @staticmethod
+    def _save_section(section, filename):
```

Comment:
Does this actually need to be a staticmethod of this class or could it go into utils?

Reply:
Could be utils.